boards/unphone8.json

```diff
@@ -0,0 +1,41 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32s3_out.ld"
```

Comment:
In the PR to the Arduino core you used a special default partition. Please specify it here as well, see an [example](https://github.com/platformio/platform-espressif32/blob/develop/boards/esp32-s3-devkitc-1.json#L5).